src/Parser/DemonHunter/Vengeance/Modules/Abilities.js

```diff
@@ -23,7 +23,8 @@ class Abilities extends CoreAbilities {
       {
         spell: SPELLS.DEMON_SPIKES,
         enabled: combatant.hasTalent(SPELLS.RAZOR_SPIKES_TALENT.id),
-        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+        buffSpellId: SPELLS.DEMON_SPIKES_BUFF.id,
+        category: Abilities.SPELL_CATEGORIES.DEFENSIVE,
         cooldown: haste => 15 / (1 + haste),
         charges: 3,
```

Comment:
Not related to the PR, but you could remove one of the demon spike entrances and replace this line with `charges: combatant.hasTalent(SPELLS.RAZOR_SPIKES_TALENT.id) ? 3 : 2,`